src/index.js

```diff
@@ -52,7 +53,9 @@ export class WT {
     this.emitter = new EventEmitter();
     this.wtConfig = { cookies: { expires: EXPIRES_IN_DAYS } };
     this.context = context;
-    this.paramDefaults = {};
+    this.paramDefaults = {
+      wt_version: WT_VERSION,
```

Comment:
I like this! will it go into `event.metadata`?

Reply:
i'm not sure that i like this (as is); recording this for every event seems excessive, _especially_ if it's going into an already unstructured metadata column. what's the motivation for tracking this/do we need it on a data level/do we need it for every event?

if we do keep this i'd suggest renaming to `wt_client_version` for accuracy.

Reply:
I don't think we have to worry about storage size, but I hear you. I actually think having app version info would be more useful (so not super opinionated on whether we leave wt version info in or not), but that's outside of the scope here.

Reply:
yeah, this will be saved in `event.metadata`. this was to help me QA that the new events were saving properly on macallan so I don't feel too strongly about keeping it. i feel ok with removing it because it doesn't add much value outside of the QA use case.